src/config/geneneralconf.cpp

```diff
@@ -49,6 +53,14 @@ void GeneneralConf::updateComponents() {
     m_sysNotifications->setChecked(config.desktopNotificationValue());
     m_autostart->setChecked(config.startupLaunchValue());
     m_closeAfterCapture->setChecked(config.closeAfterScreenshotValue());
+    m_saveAfterCopy->setChecked(config.saveAfterCopyValue());
+
+    if (!config.saveAfterCopyPathValue().isEmpty()) {
+        m_savePath->setText(config.saveAfterCopyPathValue());
+    } else {
+        ConfigHandler().setSaveAfterCopyPath(
```

Comment:
It's a little weird that default save-after-copy path is set only after opening configuration window.

Reply:
Well since you need to open the configuration window to enable the feature, I thought it was a fairly reasonable place to load that field. There probably is a better place though.